### DISCUSSION TITLE
crazyflie-lib-python examples

### DISCUSSION THREAD

**Discussion by ntukenmez3** [bitcraze/discussions#990] [Q&A]
How can we use crazyflie-lib-python examples with ros2?

**Comment by knmcguire** [comment]
Not like this no. There is no ROS2 examples ready available, so you'll need to write a ROS wrapper for them or use [crazyswarm2 ](https://imrclab.github.io/crazyswarm2/)

**Reply by ntukenmez3** [reply]
Do you have any guide showing how to write a ROS2 wrapper, that would be helpful for us.

**Reply by krichardsson** [reply]
No there is unfortunately no such guide

**Comment by Pankaj-Str** [comment]
```python
import cflib

def on_found_link_uri(link_uri):
    print(f"Found Crazyflie with URI: {link_uri}")
    cf = cflib.Crazyflie()
    cf.open_link(link_uri)
    # Now you can control the Crazyflie using the cf object
    # For example, you can set thrust or send commands.

with cflib.CrazyradioServer() as crtp_server:
    crtp_server.scan([b"Crazyflie"])
    crtp_server.wait(1.0)


```

**Reply by ntukenmez3** [reply]
My question was about cflib examples, could you clarify your answer if possible for you, thanks.

**Reply by knmcguire** [reply]
My chatgpt senses are high with this one 😉 

And also it missed the point totally with your question.

**Reply by ntukenmez3** [reply]
I completely agree with you on that point☺️
